pkg/osvscanner/osvscanner.go

```diff
@@ -599,27 +633,109 @@ func DoScan(actions ScannerActions, r reporter.Reporter) (models.VulnerabilityRe
 	}
 
 	// if vulnerability exists it should return error
-	if len(vulnerabilityResults.Results) > 0 {
+	if len(results.Results) > 0 {
 		// If any vulnerabilities are called, then we return VulnerabilitiesFoundErr
-		for _, vf := range vulnerabilityResults.Flatten() {
+		for _, vf := range results.Flatten() {
 			if vf.GroupInfo.IsCalled() {
-				return vulnerabilityResults, VulnerabilitiesFoundErr
+				return results, VulnerabilitiesFoundErr
 			}
 		}
 		// Otherwise return OnlyUncalledVulnerabilitiesFoundErr
-		return vulnerabilityResults, OnlyUncalledVulnerabilitiesFoundErr
+		return results, OnlyUncalledVulnerabilitiesFoundErr
 	}
 
-	return vulnerabilityResults, nil
+	return results, nil
+}
+
+const DepsdevURL = "api.deps.dev:443"
```

Comment:
Maybe this license fetching function and variables can go in a file pkg/depsdev/license.go?

Reply:
Moved to pkg/depsdev/license.go